app/src/main/res/values-sv/strings.xml

```diff
@@ -23,4 +23,10 @@
     <string name="missing_name">Du har inte gett ditt djur ett namn!</string>
     <string name="missing_question">Du har inte skrivit en fråga!</string>
 
+    <string name="a_dog">Missing Translations</string>
+    <string name="dog_question">Missing Translations</string>
+
+    <string name="a_pig">Missing Translations</string>
+    <string name="pig_question">Missing Translations</string>
```

Comment:
The pig strings are actually the same as the ones for `learn_question`, I'll get back to you about the dog strings.